lib/cfn-model/transforms/serverless.rb

```diff
@@ -104,9 +111,10 @@ def lambda_service_can_assume_role
 
       # Return the hash structure of the 'FunctionNameRole'
       # AWS::IAM::Role resource as created by Serverless transform
-      def function_name_role
+      def function_name_role(with_line_numbers)
+        type = with_line_numbers ? { 'value' => 'AWS::IAM::Role', 'line' => '---' } : 'AWS::IAM::Role'
```

Comment:
roll this guy into a function so it can be reused across the functions

Reply:
Updated in latest commit.